apps/shared/avifjpeg.c

```diff
@@ -254,6 +275,11 @@ avifBool avifJPEGRead(const char * inputFilename,
     avifRGBImage rgb;
     memset(&rgb, 0, sizeof(avifRGBImage));
 
+    // Standard XMP segment followed by all extended XMP segments.
+    avifRWData totalXMP = { NULL, 0 };
+    // Each byte set to AVIF_FALSE is a missing byte. Each byte set to AVIF_TRUE was read and copied to totalXMP.
```

Comment:
Nit: 0 and 1 (as opposed to AVIF_FALSE and AVIF_TRUE) are used in line 463 and line 477.